server/server.go

```diff
@@ -47,8 +48,16 @@ func ListenAndServe(ctx context.Context, httpsPort, httpPort int, handler http.H
 		opts.TLSListenerConfig.TLSConfig = &tls.Config{}
 	}
 
-	logger := logrus.StandardLogger()
-	errorLog := log.New(logger.WriterLevel(logrus.DebugLevel), "", log.LstdFlags)
+	writer := logrus.StandardLogger().WriterLevel(logrus.DebugLevel)
```

Comment:
nit: Maybe adding a comment about this being there for backwards compatibility reason